updates/commands.go

```diff
@@ -119,6 +123,111 @@ func createStatusMessage(user users.User, chatId int64, msg tgbotapi.MessageConf
 	return msg
 }
 
+func handleMonthlySummaryCommand(fatBotUpdate FatBotUpdate) {
+	update := fatBotUpdate.Update
+	bot := fatBotUpdate.Bot
+	originalChatId := update.FromChat().ID
+	user, err := users.GetUserFromMessage(update.Message)
+	if err != nil {
+		log.Error(err)
+		sentry.CaptureException(err)
+		return
+	}
+
+	if user.ID == 0 {
+		return
+	}
+	months := 6 // until we'll let the user enter the number of months
+	start, end := getDuration(months)
+	chatIds, err := user.GetChatIds()
+	if err != nil {
+		log.Error(err)
+		sentry.CaptureException(err)
+		return
+	}
+	if len(chatIds) > 0 {
+		for _, chatId := range chatIds {
+			createUserMonthlySummaryChart(bot, originalChatId, user, chatId, start, end)
+		}
+	} else {
+		log.Errorf("No chat IDs found for user %s", user.Username)
+	}
+}
+func createUserMonthlySummaryChart(bot *tgbotapi.BotAPI, originalChatId int64, user users.User, chatId int64, start, end time.Time) {
```

Comment:
There are *many* arguments sent to this function, ideally it should be more than 2,3 for clean code. Otherwise this suggests certain bits of code can be moved around.

Also, it's conventional in Go to group same arguments together, so that you can redefine it like so:
```golang
(bot *tgbotapi.BotAPI, originalChatId, chatId int64, user users.User, start, end time.Time)
```